context/app/routes_workspaces.py

```diff
@@ -16,11 +16,21 @@ def index():
     )
 
 
-@blueprint.route('/workspaces/<workspace_id>')
+@blueprint.route('/workspaces/start/<workspace_id>')
 def please_wait(workspace_id):
     flask_data = {**get_default_flask_data()}
     return render_template(
         'base-pages/react-content.html',
         flask_data=flask_data,
         title='Please Wait'
     )
+
+
+@blueprint.route('/workspaces/<workspace_id>')
+def workspace_detail(workspace_id):
+    flask_data = {**get_default_flask_data()}
+    return render_template(
+        'base-pages/react-content.html',
+        flask_data=flask_data,
+        title='Workspace'
```

Comment:
We should consider setting the current workspace name as the page title once it's loaded on the front-end - this can be an additional improvement 👍🏻